integration/SimpleCacheClient.test.ts

```diff
@@ -39,10 +39,22 @@ const removeSystemCredentials = () => {
   });
 };
 
+const deleteExistingCache = async (
+  momento: SimpleCacheClient,
+  cacheName: string
+) => {
+  const listCacheResp = await momento.listCaches();
+  const cacheList = listCacheResp.getCaches().map(c => c.getName());
+  if (cacheList.includes(cacheName)) {
+    await momento.deleteCache(cacheName);
```

Comment:
Since we now have limits on control plane api calls, you can just surround the delete call with a try catch and swallow the `CacheDoesntExist` error